src/styles/card.scss

```diff
@@ -0,0 +1,130 @@
+@import '../styles/utils/variables';
+
+.product-card {
+  border: $border-styles;
+  border-radius: 5px;
+
+  width: 200px;
+  height: auto;
+  box-sizing: border-box;
+  padding: 32px 19px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    padding-top: 40px;
+  }
+
+  &__title {
+    color: $product-info-color;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+
+    width: 166px;
+    height: 36px;
+  }
+
+  &__price {
+    width: 166px;
+    height: 18px;
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+  }
+
+  &__price-text {
+    color: #616070;
+
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+  }
+
+  &__price-value {
+    color: $product-info-color;
+
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+  }
+
+  &__code {
+    color: #616070;
+
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+  }
+
+  &__reviews-count {
+    color: $product-info-color;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__buy-button {
+    background: $theme-color;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    align-content: center;
+    text-align: center;
+    color: $default-button-color;
+    text-decoration: none;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    margin-top: 16px;
+
+    &:hover {
+      background-color: $button-hovered-bg-color;
+      color: $theme-color;
+      border: $border-hovered-styles;
+    }
+  }
+}
+
+.item {
```

Comment:
The class `.item` and its nested elements might conflict with `.product-card` if they are used in the same context. Consider using more specific class names to avoid potential conflicts.